decorators/trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    if @nameable.correct_name.length > 10
+      "#{@nameable.correct_name[0...10]}.."
```

Comment:
- Hey, since we want the name to have 10 characters, then we should make it in this range `@nameable.correct_name[0..9]` as yours currently will return 11 characters.